docs/reference/analysis/tokenizers/pathhierarchy-tokenizer.asciidoc

```diff
@@ -170,3 +170,9 @@ If we were to set `reverse` to `true`, it would produce the following:
 ---------------------------
 [ one/two/three/, two/three/, three/ ]
 ---------------------------
+
+[float]
+=== Detailed Examples
+See detailed  <<analysis-pathhierarchy-tokenizer-examples, examples here>>:: 
+
+include::tokenizers/pathhierarchy-tokenizer-examples.asciidoc[]
```

Comment:
```suggestion

```

I believe this `include::` statement is causing the build to fail. The path is relative here so you'd need to remove `tokenizers` for it to work.

However, you already link to the examples so including it here is a bit redundant. I suggest removing it all together.